src/main.js

```diff
@@ -6,7 +6,9 @@ require('prototype_creep_move');
 require('prototype_roomPosition');
 require('prototype_room_init');
 require('prototype_room_costmatrix');
-require('visualizer');
+require('screepsplus');
+
+var Stats = require('stats');
```

Comment:
I think we should adding it to `brain` is better than storing it in a local variable. Main motivation for that is simplifying the minification.
Would like to be able to minify the code for bot release. Also the are rumors that it saves CPU on the live server to have only one file.